src/contracts/interfaces/IRewardsCoordinator.sol

```diff
@@ -60,6 +70,42 @@ interface IRewardsCoordinator {
         uint32 duration;
     }
 
+    /**
+     * Sliding Window for valid RewardsSubmission startTimestamp
+     *
+     * Scenario A: GENESIS_REWARDS_TIMESTAMP IS WITHIN RANGE
+     *         <--------MAX_RETROACTIVE_LENGTH--------> t (block.timestamp)
+     *             <--valid range for startTimestamp--
+     *             ^
+     *         GENESIS_REWARDS_TIMESTAMP
+     *
+     *
+     * Scenario B: GENESIS_REWARDS_TIMESTAMP IS OUT OF RANGE
+     *         <--------MAX_RETROACTIVE_LENGTH--------> t (block.timestamp)
+     *         <----valid range for startTimestamp----
+     *     ^
+     * GENESIS_REWARDS_TIMESTAMP
```

Comment:
`GENESIS_REWARDS_TIMESTAMP` is never in range?

Reply:
What do you mean?

Reply:
For the performance based rewards it's not possible to submit a reward that starts at `GENESIS_REWARDS_TIMESTAMP` (ie. scenario B)

Reply:
Well you can if `MAX_RETROACTIVE_LENGTH` becomes a modifiable variable in the future. But I agree as of now it's theoretically not possible.

Reply:
Keeping `GENESIS_REWARDS_TIMESTAMP` in the check incase `MAX_RETROACTIVE_LENGTH` becomes modifiable in the future and we get a long enough span into the past.